src/phoneField/__snapshots__/PhoneField.unit.tsx.snap

```diff
@@ -60,8 +60,9 @@ exports[`PhoneField should have the expected markup in the DOM with custom setti
   outline: 0;
   box-shadow: none;
   border: none;
+  border-radius: 16px;
   cursor: pointer;
-  z-index: 1;
+  z-index: 0;
```

Comment:
Need the same deep position on the `select` as the arrow icon. Otherwise, in case of error display the `select` red background hide the icon.